database/src/migrations/release.0.29/biohub.sql

```diff
@@ -2218,7 +2218,7 @@ CREATE TABLE survey_summary_detail(
     parameter                       varchar(100)      NOT NULL,
     stratum                         varchar(100)      NOT NULL,
     parameter_value                 numeric(10, 0),
-    parameter_esitmate              numeric(10, 0),
+    parameter_estimate              numeric(10, 0),
```

Comment:
I've mentioned in Charlies PR to fix this, so we can probably revert all of these migration changes from this PR (just to avoid any big merge conflicts). 

Comment on Charlies PR: https://github.com/bcgov/biohubbc/pull/561/files#r720373960

Reply:
kk